apps/antalmanac/src/components/RightPane/SectionTable/GradesPopup.tsx

```diff
@@ -134,7 +137,8 @@ function GradesPopup(props: GradesPopupProps) {
                     <BarChart data={gradeData.grades} style={{ cursor: 'pointer' }}>
                         <CartesianGrid strokeDasharray="3 3" />
                         <XAxis dataKey="name" tick={{ fontSize: 12, fill: axisColor }} />
-                        <YAxis tick={{ fontSize: 12, fill: axisColor }} width={40} />
+                        <YAxis tick={{ fontSize: 12, fill: axisColor }} width={40} unit="%" />
+                        <Tooltip content={<GradeTooltip />} position={{ y: 100 }} offset={-5} />
```

Comment:
`<GradeTooltip />` is missing props. I'm curious how its still acquiring the correct values, but regardless I'd like to properly pass it props.